Source/WarpX.H

```diff
@@ -756,6 +756,41 @@ public:
         bool const step_after_start = (step >= start_moving_window_step);
         return do_moving_window && step_before_end && step_after_start;
     }
+    /** Calculate the shift per step for the moving window
+     *
+     * This function calculates how much the moving window shifts in one timestep.
+     * This is the same calculation used in MoveWindow.
```

Comment:
This function **is** used in `MoveWindow`, so it is confusing to say that this is the "same" calculation used in MoveWindow.